api/src/express-server.ts

```diff
@@ -20,12 +21,15 @@ export class ExpressServer {
     this.app = express()
     this.lbApp = new ApiApplication(options)
 
-    // Middleware migrated from LoopBack 3
     this.app.use(compression())
     this.app.use(cors())
     this.app.use(helmet())
 
-    this.app.use('/api', this.lbApp.requestHandler)
+    this.app.use(accessTokenMiddleware)
+
+    this.app.get('/api/crew-summary', function (_req: Request, res: Response) {
+      res.send('crew summary')
+    })
```

Comment:
I was going to write this endpoint, but I decided to let you. There is already a decent amount of context on how I want to set up our folders in the `/accessToken` folder.

This is the route for the new endpoint you will be creating. You will need to create folders for `users` and other models with drizzle configurations, and then you can use drizzle to query the db and construct your response.